src/ECU.cpp

```diff
@@ -165,26 +165,26 @@ void ECU::route(SensorData* data) {
 ////////////UPDATE FUNCTIONS////////////////
 ////////////////////////////////////////////
 
-void ECU::updateThrottle(SensorData* msg) {
-    if(msg->getId() == THROTTLE1_ID) {
-        throttle.setThrottle1(msg->getData());
+void ECU::updateThrottle() {
```

Comment:
Instantiation is correct, but you don't need to specify 8 - it's the default. You can just use this:
`BufferPacker unpacker(rmsg.buf);`

Reply:
Argument list for class template "BufferPacker" is missing when 8 is missing